lxd/migration/migration_volumes.go

```diff
@@ -39,6 +39,7 @@ type VolumeTargetArgs struct {
 	Refresh       bool
 	Live          bool
 	VolumeSize    int64
+	ContentType   string
```

Comment:
@monstermunchkin is there a reason this cant be of type storageDrivers.ContentType?

Reply:
If I change it to `storageDrivers.ContentType`, it creates a circular dependency.

Reply:
@monstermunchkin ah, well thats a good reason for string then :)